repo/packages/R/riak/1/command.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@tamarrow Thank you for direction. Will investigate it and may be have some questions.